rows.go

```diff
@@ -106,6 +106,10 @@ func (rows *snowflakeRows) ColumnTypePrecisionScale(index int) (precision, scale
 	switch rows.RowType[index].Type {
 	case "fixed":
 		return rows.RowType[index].Precision, rows.RowType[index].Scale, true
+	case "time":
+		return rows.RowType[index].Scale, 0, true
```

Comment:
We are setting the precision as the scale and the scale as 0, which is a little confusing to me. Let's hold the change for now and confirm with the customer what is the value he will be looking at.